astroid/nodes/node_classes.py

```diff
@@ -766,6 +766,7 @@ class Arguments(mixins.AssignTypeMixin, NodeNG):
     )
 
     _other_fields = ("vararg", "kwarg")
+    assigned_stmts: Callable
```

Comment:
Ideally, we should be more specific.

All the `assigned_stmts` method have basically the same signature. Maybe we can figure out the actual types for them.
```py
def xxx_assigned_stmts(self, node=None, context=None, assign_path=None):
    ...
```